### PR TITLE
New dependencies added to the bare metal route install guide

### PR DESCRIPTION
**Pull request by MrAlfabet**
Added Celery and (I needed this even on earlier versions of paperless-ngx) default-libmysqlclient-dev

## Proposed change

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (please explain)

## Checklist:

- [ ] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [ ] I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [ ] I have made corresponding changes to the documentation as needed.
- [ ] I have checked my modifications for any breaking changes.

### CODE REVIEW

**Comment by shamoon** [comment]
Thanks. `default-libmysqlclient-dev` Is already in both places

File: docs/setup.md [hunk 1]
```diff
@@ -266,11 +266,13 @@ supported.
     - `mime-support` for mime type detection
     - `libzbar0` for barcode detection
     - `poppler-utils` for barcode detection
+    - `default-libmysqlclient-dev` for DB communication
```
Comment (shamoon) [inline docs/setup.md:269]:
```suggestion
```

File: docs/setup.md [hunk 2]
```diff
@@ -266,11 +266,13 @@ supported.
...
+    - `celery` for spawning workers
 
     Use this list for your preferred package management:
 
     ```
-    python3 python3-pip python3-dev imagemagick fonts-liberation gnupg libpq-dev default-libmysqlclient-dev libmagic-dev mime-support libzbar0 poppler-utils
+    python3 python3-pip python3-dev imagemagick fonts-liberation gnupg libpq-dev default-libmysqlclient-dev libmagic-dev mime-support libzbar0 poppler-utils default-libmysqlclient-dev celery
```
Comment (shamoon) [inline docs/setup.md:275]:
```suggestion
    python3 python3-pip python3-dev imagemagick fonts-liberation gnupg libpq-dev default-libmysqlclient-dev libmagic-dev mime-support libzbar0 poppler-utils celery
```

**Review by shamoon — CHANGES_REQUESTED** [review]

**Review by shamoon — APPROVED** [review]

**Comment by stumpylog** [comment]
I don't think celery belongs here.  it should be installed via `pip`, not the system package manager.

**Comment by MrAlfabet** [comment]
That's what I figured as well, but that didn't work for me.

**Comment by stumpylog** [comment]
Probably something similar to step 11 then.  The actual executable location will vary depending on how and what installed it.  That is something which could be documented instead.

**Review by stumpylog — CHANGES_REQUESTED** [review]
If you'd like to add a note somewhere regarding the possible location of the celery executable, similar to the one about gunicorn, that could be useful for those using bare metal.

**Comment by shamoon** [comment]
Thanks for clarifying I didn't notice. @MrAlfabet Please feel free to submit a new PR with that doc change

**Pull request closed without merging**